tests/test.js

```diff
@@ -20,11 +20,12 @@ describe('when parsing an Ogg Stream', function(){
             .on('data', function(t){
                 tags[t.type] = t.value;
             })
-            .on('end', function(){
+            .on('finish', function(){
```

Comment:
This seems to have changed at some point since 0.10, and is how I have to use the stream in my other code as well. Maybe move to using `readable-stream` as the base to make it version-agnostic?